pallets/nfts/src/lib.rs

```diff
@@ -693,6 +736,24 @@ pub mod pallet {
 		CollectionNotEmpty,
 		/// The witness data should be provided.
 		WitnessRequired,
+		/// The account owns zero items in the collection.
+		NoItemOwned,
+		/// The collection has existing approvals.
+		CollectionApprovalsExist,
```

Comment:
Note: Tested in 

- destroy_works: https://github.com/r0gue-io/pop-node/pull/387/files#r1881258572